Java.md

```diff
@@ -1 +1,201 @@
-# Java
\ No newline at end of file
+# Java
+
+## Class
+
+### Layout
+
+Methods and fields should be organized as below:
+
+```java
+class MySuperbClass {
+	// Fields
+	
+	// Constructors
+	
+	// Private & protected methods
+	
+	// Public methods
+}
+```
+
+### Indentation
+
+Standard indentation should be **4 soft spaces**. When a carriage return is needed, keep same indentation.
```

Comment:
I always find the term `soft tabs` and `hard tabs` confusing. Can we just say **4 spaces (_no tabs_)** to be clearer?